### PR TITLE
feat: add onekey support

### PR DESCRIPTION
**Pull request by loatheb**
Hi team:
I have added information related to OneKey, OneKey also provides cross-chain software wallet and hardware wallet. Here is the official website https://onekey.so. 

I have improved and supplemented the content with reference to other wallet formats, and I hope everyone can give a review. Thanks.

### CODE REVIEW

File: _data/compatibility/onekey.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    version: "4.2.1"
+  features:
+    receive:
+      p2sh_wrapped: "true"
+      bech32: "true"
+      default: "p2sh_wrapped"
+    send:
+      bech32: "true"
+      change_bech32: "true"
+      segwit_v1: "Fails on validation of the address."
```
Comment (bitschmidty) [inline _data/compatibility/onekey.yaml:55]: @loatheb does onekey allow sending to taproot addresses?

File: _data/compatibility/onekey.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      bech32_p2wsh: "true"
+  examples:
+    - image: /img/compatibility/onekey/segwit/receive-screen.png
+      caption: >
+        By default, OneKey generates a BIP49 P2SH-P2WPKH Base58 receive addresses.
+        There is also an option to generate a BIP44 P2PKH、BIP84 P2WPKH and BIP86 P2TR addresses.
```
Comment (bitschmidty) [inline _data/compatibility/onekey.yaml:61]:
```suggestion
        There is also an option to generate a BIP44 P2PKH, BIP84 P2WPKH and BIP86 P2TR addresses.
```

**Review by bitschmidty — COMMENTED** [review]
@loatheb thanks for this contribution! One suggestion and one question

File: _data/compatibility/onekey.yaml (hunk 1 above)
Comment (loatheb) [reply _data/compatibility/onekey.yaml:55]: Sorry I missed this part, I just tried OneKey to support it.

https://btc.com/5859760124a47c529c8128e35675f44efeef69f9841aa1a04bf9d759c641f3d7

image

File: _data/compatibility/onekey.yaml (hunk 2 above)
Comment (loatheb) [reply _data/compatibility/onekey.yaml:61]: Sorry for the late reply, it has been updated, thank you very much.

**Review by bitschmidty — APPROVED** [review]
ACK, thank you @loatheb !

**Pull request merged**